ui/src/environments/environment-type.ts

```diff
@@ -41,6 +41,8 @@ export interface Environment {
   // Example value: https://aousupporthelp.zendesk.com/hc/
   zendeskHelpCenterUrl: string;
 
+  inactivityTimeoutInSeconds: number;
```

Comment:
opt: personally I prefer to just directly concatenate the units on the normal variable name, rather than using "in" e.g. inactivityTimeoutSeconds